src/subscription.ts

```diff
@@ -1064,7 +1087,7 @@ export class Subscription extends EventEmitter {
    * @private
    */
   private _listen(): void {
-    this.on('newListener', event => {
+    this.on('newListener', (event: string) => {
```

Comment:
This was my hint above that something was amiss.  I would expect the types to just figure out this is a string.

Reply:
Yeah. Previously, `on()` was basically typed as Ben suggested below, since it's inherited from `EventEmitter`.

Reply:
Still don't think we want to require the typing here.